packager/media/replicator/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 Google Inc. All rights reserved.
```

Comment:
nit: Google LLC

We recently had to do a mass update to correct this in old files, so let's not add the old name in any new files.  Thanks!

Reply:
This is great - I made the same mistake you did a few weeks ago 😅